### PR TITLE
JCLOUDS-1370: Add CannedAccessPolicy constants

### PR DESCRIPTION
**Pull request by gaul**
Also use CaseFormat instead of extra logic.

### CODE REVIEW

File: apis/s3/src/main/java/org/jclouds/s3/domain/CannedAccessPolicy.java [hunk 1]
```diff
@@ -36,29 +38,50 @@
     /**
      * Owner gets FULL_CONTROL. No one else has access rights (default).
      */
-    PRIVATE("private"),
```
Comment (stevegomez17) [inline apis/s3/src/main/java/org/jclouds/s3/domain/CannedAccessPolicy.java]: “PRIVATE”

File: apis/s3/src/main/java/org/jclouds/s3/domain/CannedAccessPolicy.java [hunk 2]
```diff
@@ -36,29 +38,50 @@
...
+    PRIVATE,
     /**
      * Owner gets FULL_CONTROL and the anonymous identity is granted READ
      * access. If this policy is used on an object, it can be read from a
      * browser with no authentication.
      */
-    PUBLIC_READ("public-read"),
```
Comment (stevegomez17) [inline apis/s3/src/main/java/org/jclouds/s3/domain/CannedAccessPolicy.java]: “PUBLIC _READ”

File: apis/s3/src/main/java/org/jclouds/s3/domain/CannedAccessPolicy.java [hunk 3]
```diff
@@ -36,29 +38,50 @@
...
+    PUBLIC_READ,
     /**
      * Owner gets FULL_CONTROL, the anonymous identity is granted READ and
      * WRITE access. This can be a useful policy to apply to a bucket, but is
      * generally not recommended.
      */
```
Comment (stevegomez17) [inline apis/s3/src/main/java/org/jclouds/s3/domain/CannedAccessPolicy.java]: PUBLIC_READ

Comment (gaul) [reply apis/s3/src/main/java/org/jclouds/s3/domain/CannedAccessPolicy.java]: I don't understand any of your comments.  Can you use complete sentences?

**Pull request merged**